aptcacher/server.sls

```diff
@@ -24,5 +24,24 @@ aptcacher_services:
   - names: {{ server.services }}
   - watch:
     - file: /etc/apt-cacher-ng/acng.conf
+    - module: aptcacher_limits_override
+
+aptcacher_limits_override_dir:
+  file.directory:
+  - name: /etc/systemd/system/apt-cacher-ng.service.d/
+  - mode: 755
+  - makedirs: true
+
+aptcacher_limits_override:
+  file.managed:
+  - name: /etc/systemd/system/apt-cacher-ng.service.d/override.conf
+  - source: salt://aptcacher/files/systemd_override.conf
+  - template: jinja
+  - require:
+    - file: aptcacher_limits_override_dir
```

Comment:
There should be a watch_in on service.
Also makedirs could be here so dir resource above is not needed.